api-operator/pkg/config/system.go

```diff
@@ -43,3 +45,31 @@ func SetOperatorNamespace() {
 		OperatorNamespace = ns
 	}
 }
+
+// Checks whether operator is used cluster wide or namespace wide.
+func getWatchClusterLevel() (watchClusterLevel bool) {
```

Comment:
Shall we start the Godoc comment with the function name? the convention for Godoc is to start the comment with its declaration,
Ref: https://blog.golang.org/godoc

Reply:
Fixed. Thank you for pointing this.